docs/rules/no-computed-in-data.md

```diff
@@ -0,0 +1,81 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/no-computed-in-data
+description: require computed properties are not used in the data()
+---
+# vue/no-computed-in-data
+> require computed properties are not used in the data()
+
+- :gear: This rule is included in `"plugin:vue/strongly-recommended"` and `"plugin:vue/recommended"`.
+
+> require computed properties are not used in the data()
+
+- :gear: This rule is included in `"plugin:vue/strongly-recommended"` and `"plugin:vue/recommended"`.
+
+Please describe the origin of the rule here.
+
+## :book: Rule Details
+
+This rule report computed properties are used in data property
+
+Examples of **incorrect** code for this rule:
```

Comment:
Could you rewrite a document like any other document?

`Examples of ** incorrect ** code for this rule:`
Is no longer used.